### ISSUE TITLE
Workflow improvements

### ISSUE THREAD

**Issue by gothub** [DataONEorg/rdataone#287]
Update the rdataone and dependent packages to enhance ease-of-use and to reduce the amount of user effort and to reduce the amount of effort required to achieve a desired outcome.

**Comment by amoeba** [comment]
Getting the data for a given Solr result is a great example of a workflow we might target with this work. See [this Discussion](https://github.com/DataONEorg/dataone/discussions/22), specifically the answer, how much explanation it takes, and how the answer is still incomplete despite its length.

**Comment by mbjones** [comment]
Agreed. That said, in the case of the "download all objects for a package" case, that is what `getPackage` was designed for, which should significantly shorten the steps needed (e.g., add the `resourceMap` field to the SOLR query, and then use it to call `/packages/bagit-1.0.0/{resource_map_pid}` on the MN. Two majro barriers with this: 1) can't be called on the CN (I think), and 2) as an optional API, only a few repositories implement this API (namely Metacat and GMN nodes).
